partEZ/app/Http/Controllers/EventController.php

```diff
@@ -379,10 +379,16 @@ public function splitEmails($eid)
     {
         $input = Request::all();
         $emails = $input['email-list'];
-
         $emails = array_map('trim', explode(',', $emails));
         self::inviteUsers($emails, $eid);
-        return view('events/success_event');
```

Comment:
Why was this removed? Isn't the regular splitEmails still called for non-public emails?